### PR TITLE
[doc] Improve the punctuation in the documentation of the module atexit

### PR DESCRIPTION
**Pull request by geryogam**
This PR will:

- add a comma to a hardly parsable sentence;
- uniformise quotation marks.

### CODE REVIEW

File: Doc/library/atexit.rst [hunk 1]
```diff
@@ -83,21 +83,22 @@ making an explicit call into this module at termination. ::
...
+           outfile.write('%d' % _count)
 
    import atexit
+
    atexit.register(savecounter)
 
 Positional and keyword arguments may also be passed to :func:`register` to be
 passed along to the registered function when it is called::
 
    def goodbye(name, adjective):
```
Comment (iritkatriel) [inline Doc/library/atexit.rst:96]: shouldn't these be kwargs for the example below to work?

Comment (geryogam) [reply Doc/library/atexit.rst:96]: @iritkatriel I have just tried it on Python 3.9.2 and both positional and keyword arguments work.

**Review by iritkatriel — APPROVED** [review]

**Comment by maggyero** [comment]
Thank you for reviewing this @iritkatriel.

**Comment by iritkatriel** [comment]
closing and reopening to kick the buildbot.

**Pull request merged**